### PR TITLE
Feat/phonenumber convention sync

### PR DESCRIPTION
**Pull request by Dachamelemon**
OP-1464

### CODE REVIEW

File: app/modifiers/inputmaskTel.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { modifier } from 'ember-modifier';
+import Inputmask from 'inputmask';
+
+export default modifier(function inputmaskTel(element) {
+  new Inputmask({
+    regex: '(\\+)\\d{6}\\d*',
```
Comment (claire-lovisa) [inline app/modifiers/inputmaskTel.js:6]: Shouldn't it be 10 instead of 6 ? For the size of a regular belgian phone number

**Review by claire-lovisa — CHANGES_REQUESTED** [review]
I found an issue I didn't see the first time. If you fill in a number, save the bestuur, then try to re-edit the same page and immediately save, an error will pop up on the phone number saying ` Enkel een plusteken en cijfers zijn toegelaten` even though the number seem to be filled in and formatted correctly

**Comment by mirdono** [comment]
> I found an issue I didn't see the first time. If you fill in a number, save the bestuur, then try to re-edit the same page and immediately save, an error will pop up on the phone number saying ` Enkel een plusteken en cijfers zijn toegelaten` even though the number seem to be filled in and formatted correctly

The same behaviour seems to be present when trying to save an existing adm. unit location that already had a phone number. If you edit all present phone numbers before saving no such error occurs.

File: app/controllers/administrative-units/administrative-unit/core-data/edit.js [hunk 2]
```diff
@@ -150,6 +150,10 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataEditController
       structuredIdentifierKBO,
     } = this.model;
 
+    yield Promise.all([contact]);
+
+    console.log(contact.telephone);
```
Comment (claire-lovisa) [inline app/controllers/administrative-units/administrative-unit/core-data/edit.js]: This can probably go away ? :)

**Review by claire-lovisa — COMMENTED** [review]
The bug seems fixed ! :) Just need to wait for the backend PR to be ready to merge both at the same time !

**Review by claire-lovisa — APPROVED** [review]

**Pull request merged**